napari/_qt/qt_resources/styles/00_base.qss

```diff
@@ -43,6 +43,13 @@ QWidget[emphasized="true"] > QFrame {
     background-color: {{ foreground }};
 }
 
+QWidget:disabled {
+  background-color: {{ background }};
+  selection-background-color: transparent;
+  border: 1px solid;
+  border-color: {{ foreground }};
+  color: {{ opacity(text, 90) }};
+}
```

Comment:
```suggestion
}

QModelMenu::separator {
	height: 1 px;
	background: {{ opacity(text, 90) }};
	margin-left: 17 px;
	margin-right: 6 px;
	margin-top: 5 px;
	margin-bottom: 3 px;
}

QModelMenu {
	padding: 6 px;
}
```

What do you think of something like this to make the separator lines a bit more distinct on dark mode and subtle on light?
Plus give the top and bottom a bit more breathing room?

Reply:
Here's the look:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/76622105/205484177-b9e1fcba-d63e-4e3e-8ff4-7875a41a87b3.png">
<img width="252" alt="image" src="https://user-images.githubusercontent.com/76622105/205484195-32579220-e775-46af-8bd5-ca16c83fbead.png">

Reply:
Yep that looks way better :+1: Thank you for the follow up and style suggestions!

Just in case, a preview on Windows (will update the previous on the OP):

![ctx_menu_after_suggestion](https://user-images.githubusercontent.com/16781833/205702493-eb9c3473-3ef9-42b8-b74f-7e2223113369.png)

![ctx_menu_after_light_suggestion](https://user-images.githubusercontent.com/16781833/205702537-104e712c-5fa6-4493-9997-18c6cfcc3bee.png)